### ISSUE TITLE
win10 setup faild in hiredis

### ISSUE THREAD

**Issue by DavinciDevil** [SerpentAI/SerpentAI#132]
there is the error code : 

----------------------------------------
  Failed building wheel for hiredis
  Running setup.py clean for hiredis
Failed to build hiredis
Installing collected packages: hiredis, aioredis, mss, pymsgbox, pyscreeze, PyTweening, PyAutoGUI, pyperclip, win32core, win32gui, enum34, html5lib, bleach, markdown, protobuf, tensorflow-tensorboard, tensorflow-gpu, Keras, Kivy-Garden, Kivy, pytesseract, txaio, autobahn, editdistance
  Running setup.py install for hiredis ... error
    Complete output from command D:\ProgramFiles\Anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\user\\AppData\\Local\\Temp\\pip-build-s7xv0ehs\\hiredis\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\user\AppData\Local\Temp\pip-stlej_0e-record\install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.6
    creating build\lib.win-amd64-3.6\hiredis
    copying hiredis\version.py -> build\lib.win-amd64-3.6\hiredis
    copying hiredis\__init__.py -> build\lib.win-amd64-3.6\hiredis
    running build_clib
    building 'hiredis_for_hiredis_py' library
    creating build\temp.win-amd64-3.6
    creating build\temp.win-amd64-3.6\vendor
    creating build\temp.win-amd64-3.6\vendor\hiredis
    C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.10240.0\ucrt" /Tcvendor/hiredis/read.c /Fobuild\temp.win-amd64-3.6\vendor/hiredis/read.obj
    read.c
    vendor/hiredis/read.c(122): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/read.c(182): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/read.c(261): warning C4267: '=': conversion from 'size_t' to 'unsigned long', possible loss of data
    vendor/hiredis/read.c(262): warning C4244: '=': conversion from '__int64' to 'long', possible loss of data
    vendor/hiredis/read.c(317): warning C4244: '=': conversion from '__int64' to 'long', possible loss of data
    vendor/hiredis/read.c(513): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
    C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.10240.0\ucrt" /Tcvendor/hiredis/sds.c /Fobuild\temp.win-amd64-3.6\vendor/hiredis/sds.obj
    sds.c
    vendor/hiredis/sds.c(60): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(107): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(145): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(227): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(228): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(245): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(246): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(281): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(282): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(327): warning C4267: 'return': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(356): warning C4267: 'return': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(434): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(455): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(555): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(556): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(581): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(585): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(593): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(601): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(602): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(607): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(614): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
    vendor/hiredis/sds.c(638): warning C4267: 'return': conversion from 'size_t' to 'int', possible loss of data
    C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\lib.exe build\temp.win-amd64-3.6\vendor/hiredis/read.obj build\temp.win-amd64-3.6\vendor/hiredis/sds.obj /OUT:build\temp.win-amd64-3.6\hiredis_for_hiredis_py.lib
    Microsoft (R) Library Manager Version 14.00.24210.0
    Copyright (C) Microsoft Corporation.  All rights reserved.

    LINK : MSIL .netmodule or module compiled with /GL found; restarting link with /LTCG; add /LTCG to the link command line to improve linker performance
    Microsoft (R) Library Manager Version 14.00.24210.0
    Copyright (C) Microsoft Corporation.  All rights reserved.

    running build_ext
    building 'hiredis.hiredis' extension
    creating build\temp.win-amd64-3.6\Release
    creating build\temp.win-amd64-3.6\Release\src
    C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Ivendor -ID:\ProgramFiles\Anaconda3\include -ID:\ProgramFiles\Anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.10240.0\ucrt" /Tcsrc\hiredis.c /Fobuild\temp.win-amd64-3.6\Release\src\hiredis.obj
    hiredis.c
    d:\programfiles\anaconda3\include\pyconfig.h(222): fatal error C1083: Cannot open include file: 'basetsd.h': No such file or directory
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\cl.exe' failed with exit status 2

    ----------------------------------------
Command "D:\ProgramFiles\Anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\user\\AppData\\Local\\Temp\\pip-build-s7xv0ehs\\hiredis\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\user\AppData\Local\Temp\pip-stlej_0e-record\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\user\AppData\Local\Temp\pip-build-s7xv0ehs\hiredis\

**Comment by DavinciDevil** [comment]
I have solved by rebuild my Visual C++ Build Tools 2015
